apps/docs/developer/getting-started.md

```diff
@@ -5,12 +5,123 @@
 Steps to getting a local instance running
 
 - Setup the database
-  - Method 1 (prefered): Execute Docker compose file `docker/docker-compose.yml` to prepare PostgresQL and Redis instances, and map out port 5432 and 6379 respectively by default.
+  - Method 1 (prefered): Execute Docker compose file `docker-compose.yml` to prepare PostgresQL and Redis instances, and map out port 5432 and 6379 respectively by default.
   - Method 2: Run Dev VM (download from S3 bucket - contact the Intake24 team), which maps out the database on 192.168.56.10:5432 (PostgreSQL). Start this VM.
 - Obtain up-to-date database snapshots from the intake24 team and import them into a database by scripts or DB tools (e.g. DBeaver). Please check the guidance on [importing database snapshots with DBeaver](https://dbeaver.com/docs/wiki/Backup-Restore/).
 - Local servers need to be run for `api`, `admin` and `survey` from each respective folder:
 - To start `api` / `admin` / `survey`: `pnpm dev` for live reloads
 
+### Steps to run locally
+
+#### Prerequisites
+
+- Node.js v22.12.0 or newer
+- pnpm v9.x or newer
+- Docker v29 or newer (for running PostgreSQL 17 and Redis 8 instances)
+- 4vGB+ free disk space (Or 50GB+ free disk space including downloading and extracting food images and database snapshots)
+
+#### 1. Git clone Intake24 project
+
+```sh
+git clone https://github.com/intake24/intake24.git
+```
+
+#### 2. Install node packages for the whole projects
+
+on project root folder, run
+
+```sh
+pnpm install
+```
+
+#### 3. Spin up Redis and PostgreSQL instance using docker compose
+
+on project root folder, run
+
+```sh
+docker compose up -d
+```
+
+#### 4. Initialise the CLI application
+
+on project root folder, run below command to build the CLI application under `apps/cli` folder using development configuration
+
+```sh
+pnpm cli:dev
+```
+
+This will create `.env` file under `apps/cli` folder, and complete the initial setup for the CLI application. You can quite the process after the command finishes.
+
+#### 5. Use CLI tools to create .env files for Intake24 API, Intake24 frontend (SPA), and Intake24 admin tool SPA applications, and populate system database
+
+on project root folder, run below command
+
+```sh
+pnpm cli init:env
+```
+
+This will create `.env` files under `apps/api`, `apps/admin` and `apps/survey` folders, and populate the system database with default data. You can also use `-f` flag to force overwrite existing `.env` files if any.
+
+```sh
+pnpm cli init:db:system
+```
+
+This will populate the system database with default data only. Follow the prompts to set up the admin user account.
+
+If the system database snapshot is not-to-date with the codebase, you may need to run database migrations to bring the database schema up-to-date by running below command on project root folder:
+
+```sh
+pnpm db:system:migrate
```

Comment:
add db:foods:migrate too, it will be out of sync
there is also common alias `pnpm:db:migrate` for both I think